BaseFilter/sections/general_extensions.txt

```diff
@@ -272,6 +272,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/146786
+asianage.com#%#AG_onLoad(() => { try { document.querySelector('.rightCol').innerHTML = document.querySelector('.rightCol').innerHTML.replaceAll('ADVERTISEMENT', '')} catch {} })
```

Comment:
OK.

Reply:
I prefer first version with `.rightCol`, because it will only replace content for a specific element, not the whole `body`.
Checking if element exist also would be nice, but as it's in `try...catch`, so maybe it's not required.

Reply:
>It doesn't look necessary, because there is no more ADVERTISEMENT in the content except the column.

The simple example is here - `https://www.asianage.com/search?page=search&srh=news&search=Test+ADVERTISEMENT`
It removes `ADVERTISEMENT` text from the search field.

Reply:
Pushed a commit.